presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -60,3 +60,7 @@ The following table lists the returned columns and what statistics they represen
     - The highest value found in this column
     - ``NULL`` in the table summary row. Available for columns of DATE, integer, floating-point, and fixed-precision
       data types.
+  * - ``histogram``
+    - The histogram for this column
+    - A summary of the underlying histogram is displayed in the query result for human-readable format. ``NULL`` in the table summary row.
+      Availability depends on connector support.
```

Comment:
'Availability depends on connector support.' - this is true for other statistics too, so there should not be a need to explicit about this here ?